index.d.ts

```diff
@@ -104,6 +104,7 @@ declare namespace Pusher {
 
   export interface PresenceChannelData {
     user_id: string;
+    user_info: { [key: string]: string };
```

Comment:
It's actually [`JSON.stringify`'ed](https://github.com/pusher/pusher-http-node/blob/5b098abdc5c7e885b7c069c03c8dc7e8b89725b0/lib/auth.js#L8) so the type is even less restrictive than this.
```suggestion
    user_info: Object;
```

Reply:
The `user_info` must be a map, this is enforced by the backend. As I understand, the correct type would be `{ [key: string]: Object }`.